docs/specification/objects.md

```diff
@@ -271,7 +273,7 @@ len(`name`) | <=    | 128, UTF-8 Encoded
 `mass_fraction` | <= | 1
 `volume_fraction` | <= | 1
 `number_fraction` | <= | 1
-name | must be unique | among the ingredients of process 
+name | must be unique | among the ingredients of process
```

Comment:
Would the fact that `name` and `labels` must be drawn from the `allowed_names` and `allowed_labels` sets in the process template be considered a constraint?

Reply:
In was waiting until that second issue was resolved, which it is now.